Makefile

```diff
@@ -1,5 +1,6 @@
 NAME="github.com/goto/stencil"
 VERSION=$(shell git describe --always --tags 2>/dev/null)
+PROTON_COMMIT := "<PROTON_COMMIT>"
```

Comment:
Do we want to use the actual commit here, instead of placeholder ?

Reply:
updated with actual commit.